src/memory.c

```diff
@@ -379,18 +395,4 @@ emu_read(uint8_t reg, bool debugOn)
 	}
 	if (!debugOn) printf("WARN: Invalid register %x\n", DEVICE_EMULATOR + reg);
 	return -1;
-}
-
-void cartridge_attach(void)
-{
-	size_t ret = -1;
-	size_t i = 0;
-
-	if (cartridge_file) {
-		CART = calloc(CART_SIZE, sizeof(uint8_t));
-		while (ret) {
-			ret = SDL_RWread(cartridge_file, &CART[i], 1, CART_SIZE - i);
-			i += ret;
-		}
-	}
-}
+}
```

Comment:
I think with what was removed here, `memory.h` should be updated to match, perhaps adding the prototype for the added function here too.

Reply:
Aye, cleaned up memory.h and some symbols in main.c